spring-content-s3/src/test/java/internal/org/springframework/content/s3/store/DefaultS3StoreImplTest.java

```diff
@@ -325,7 +328,7 @@ public String convert(S3ObjectId source) {
 							});
 							Context("when called with an entity", () -> {
 								BeforeEach(() -> {
-									entity = new TestEntity();
+									entity = new TestEntity("12345-67890");
```

Comment:
Here (and below) I added a contentId which is actually ignored because one of the previous commits adds behavior that if contentId is null then null is returned for the Resource immediately, without calling any converters and so the tests fail.